### ISSUE TITLE
Please make me an antique wooden cussing bear

### ISSUE THREAD

**Issue by deathbearbrown** [Wilto/To-Hew-List#1]
![tumblr_n5yrjvlweu1s71q1zo1_r2_1280](https://cloud.githubusercontent.com/assets/607541/4547886/aaa55684-4e50-11e4-9e22-86d6795c95cf.png)

I would like one with overalls. Sign to say "BUTTS"

**Comment by Wilto** [comment]
BOCOUPFEST PROJECT

**Comment by miketaylr** [comment]
Alternate sign title: https://www.youtube.com/watch?v=P4Bda6_usuc

**Comment by jorydotcom** [comment]
Can we make other things that cuss, too?

**Comment by deathbearbrown** [comment]
Of course! We can modernize the antique wooden cussin' bear tradition with other wooden cussin entities.

**Comment by jorydotcom** [comment]
OK great, can I get one large-ish Barn Owl, sign to say "Your Mom"

On Tue, Oct 7, 2014 at 6:17 PM, Sue Lockwood notifications@github.com
wrote:

> Of course! We can modernize the antique wooden cussin' bear tradition with
> other wooden cussin entities.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Wilto/To-Hew-List/issues/1#issuecomment-58272313.

## 

Jory Burson
Chief Operating Officer
http://bocoup.com | 1-617-379-2752
